java/adapter/avro/src/test/java/org/apache/arrow/AvroTestBase.java

```diff
@@ -70,7 +78,7 @@ protected Schema getSchema(String schemaName) throws Exception {
   }
 
   protected VectorSchemaRoot writeAndRead(Schema schema, List data) throws Exception {
-    File dataFile = TMP.newFile();
+    File dataFile = tmp.resolve("data").toFile();
```

Comment:
Maybe this could be updated to use in-memory streams instead of file streams? Then we could get rid of the filesystem dependency entirely

Reply:
@lidavidm Done